lineman/app/components/navbar/navbar.haml

```diff
@@ -7,9 +7,9 @@
     .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
       %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
         %i.fa.fa-lg.fa-inbox>
+          %span.badge{ng-show: 'unreadThreadCount() > 0'}
```

Comment:
Sorry just a frustrated dig at all the class names on one element after refactoring my dropdown arrow PR.